3.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (string, index) {
```

Comment:
Consider splitting this up - I recognize that this approach is concise, but I feel like it does too much in one place.  We also don't get an intention revealing identifier for this function - I just know it's in module `3`...